demo/index.html

```diff
@@ -10,8 +10,9 @@
       href="../node_modules/prismjs/themes/prism.css"
     />
     <link rel="stylesheet" href="https://unpkg.com/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
-    <link rel="stylesheet" type="text/css" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/demoWrapper.css" />
-    <link rel="stylesheet" type="text/css" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/elementDemoStyles.css" />
+    <link rel="stylesheet" href="https://unpkg.com/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/@aurodesignsystem/webcorestylesheets@latest/dist/demoWrapper.css" />
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/@aurodesignsystem/webcorestylesheets@latest/dist/elementDemoStyles.css" />
```

Comment:
Please update to mirror the following 

https://github.com/AlaskaAirlines/auro-avatar/blob/main/demo/index.html#L12-L17

Reply:
I will updated. thank